plugins/woocommerce-blocks/assets/js/blocks/checkout/inner-blocks/checkout-pickup-options-block/style.scss

```diff
@@ -24,28 +24,33 @@
 				margin-left: auto;
 			}
 		}
-		.wc-block-components-radio-control__description-group {
-			display: none;
-		}
-		.wc-block-components-radio-control__option-checked {
-			.wc-block-components-radio-control__description-group {
-				display: block;
-			}
-		}
 		.wc-block-components-radio-control__label-group {
 			em {
 				text-transform: uppercase;
 				font-style: inherit;
 			}
 		}
 		.wc-block-components-radio-control__description-group {
-			width: 100%;
+			display: block;
+			width: calc(100% + em($gap-huge));
 			box-sizing: border-box;
-			background-color: $universal-background;
 			border-radius: $universal-border-radius;
 			padding: 1px em($gap-small);
+			margin-left: -(em($gap-huge));
 			margin-top: em($gap-smaller);
 			@include font-size(regular);
+
+			.read-more-content {
+				div {
+					margin-right: em($gap-larger);
```

Comment:
Can we increase this a bit, as there are a few edge cases where the description text and the "more" button are really close together
I played with it in the browser an 3.25em seems to be about right. Not sure how that translates to the sass variables we're using though

![Screenshot 2024-05-07 at 11 00 48](https://github.com/woocommerce/woocommerce/assets/3966773/227e9422-4179-4db9-867a-a79b484e765b)

Reply:
That's for this feedback, @alexflorisca. The SASS variables are defined in https://github.com/woocommerce/woocommerce/blob/f040e3acf7df9420a09d37b84358ac7d2e03b8a3/plugins/woocommerce-blocks/assets/css/abstracts/_variables.scss#L3-L11

Initially, I was using `$gap-largest`, which translates to `48px`, but the gap was a bit too big. For that reason, I was using `$gap-larger`, which translates to `36px`. I see what you mean by adjusting the browser width. To get values between `$gap-largest` and `$gap-larger`, we can multiply `$grid-unit` with a different value, e.g.

```css
5 * $gap-smaller   // 40px
5.5 * $gap-smaller // 44px 
```

I'll check both options against various screen widths.

Reply:
@alexflorisca I had a chat with @nikkivias about this, and we agreed on the following two changes:
- Name the links `Read more` and `Read less` instead of `more` and `less`.
- Place the links below the pickup details text instead of behind it.

@nikkivias shared this design with me:

![CleanShot 2024-05-07 at 22 19 16@2x](https://github.com/woocommerce/woocommerce/assets/3323310/1c8c8df5-61ed-4127-9b9b-a1838cbd34ea)